doc/source/rllib/rllib-checkpoints-and-exports.rst

```diff
@@ -0,0 +1,346 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. _checkpoints-and-exports-docs:
+
+#################
+Using Checkpoints
```

Comment:
The more I think about this, I think we should call this whole doc

"Saving and loading your RL algorithms" 

or so (checkpoints can be mentioned right away, in subtitles etc.). It's better to propose a concrete workflow to users than to expose internal abstractions.

Reply:
+1

Reply:
done